rules/integrations/github/execution_github_ueba_multiple_behavior_alerts_from_account.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2023/12/14"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/12/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule is part of the "GitHub UEBA - Unusual Activity from Account Pack", and leverages alert data to
+determine when multiple alerts are executed by the same user in a timespan of one hour. 
+Analysts can use this to prioritize triage and response, as these alerts are a higher indicator of compromised user
+accounts or PATs.
+"""
+from = "now-60m"
+index = [".alerts-security.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "GitHub UEBA - Multiple Alerts from a GitHub Account"
+risk_score = 47
+rule_id = "929223b4-fba3-4a1c-a943-ec4716ad23ec"
+severity = "medium"
+tags = ["Domain: Cloud",
+        "Use Case: Threat Detection",
+        "Use Case: UEBA",
+        "Tactic: Execution",
+        "Rule Type: Higher-Order Rule"
+        ]
+timestamp_override = "event.ingested"
+type = "threshold"
+
+query = '''
+signal.rule.tags:("Use Case: UEBA" and "Data Source: Github") and kibana.alert.workflow_status:"open"
```

Comment:
I'd caution on using tags as part of detection logic. The tags use a specific format that is custom to TRaDE whereas upstream there is no standardization requirements. As a result, we are assuming that prebuilt or custom rules HAVE to have these tags.

As an alternative, the alert schema does carry `event.dataset` in documents within the `.alerts-security.*` data view or index patterns. As a result, we can search in alert documents for `event.dataset:github` and find the same documents. `event.dataset` is in ECS, therefore we are safe to assume this will be consistent.

Reply:
@terrancedejesus what would you suggest for the "Use Case: UEBA" tag that distinguishes which rules should be part of the threshold?

Reply:
Yes @terrancedejesus this is something we have also explored. We ended up settling (for now) on leveraging the UEBA tag in conjunction with either another tag or with (as you mention) an `event.dataset` value. There are for now as far as we know no other reliable methods of applying this filter. Unless we add a specific new field to the rule, a boolean for whether or not a rule is part of a UEBA pack.